src/views/List.css

```diff
@@ -98,19 +98,48 @@ section {
 	display: flex;
 	flex-direction: column;
 	align-items: center;
+	padding-top: 2em;
+	padding-bottom: 2em;
 }
 
 /* List Item */
+.listContainer {
+	width: 100%;
+	border: black 3px solid;
+}
+
 .listItemRow {
 	display: flex;
 	width: 100%;
 }
 
+.listItemRow input {
+	height: 2em;
```

Comment:
I would suggest not setting a height for the input, but rather allow it to be defined by the size of the font + padding. 

I also wonder if we should use the unit em at all. While I was working on updating the fonts in a previous PR (the font for the list items seems to be changed now), I had issues using em because it indicates size relative to the size of the font. We have many varying font sizes, so this would cause inconsistencies.